### DISCUSSION TITLE
Problemas con la task 03, Lab 6.02

### DISCUSSION THREAD

**Discussion by maguiargil** [rramosp/ai4eng.v1#259] [06 - MÉTODOS NO SUPERVISADOS]
Buenas noches,

Después de varias ejecuciones y revisiones de mi propuesta de solución para la task 3 del laboratorio 6.02 sigo teniendo este inconveniente:

![image](https://github.com/rramosp/ai4eng.v1/assets/55927354/13657975-eb43-4238-9a77-bea25240839b)

Quisiera que por favor me ayudaran a identificar qué puede estar saliendo mal. [La propuesta de solución es la siguiente:](https://colab.research.google.com/drive/1_0R-2g7AojV-jtsxC94M1k6jY-aV-MXr?usp=sharing)

De antemano, muchas gracias.

**Comment by Felipe-RA** [comment]
Saludos @maguiargil 

Intuyo que tu error es que estas entrenando y transformando el nmf antes de crear tus datos de entrenamiento y tu vector de entrenamiento.

Debes sólo entrenar el nmf, transformar X para obtener Xt (usando el nmf que entrenaste) y luego entrenar tu vector v para obtener vt (haciendo la transposición indicada en las instrucciones y comentarios del código)


Si sigues con problemas, me comentas y lo miramos.

¡Hasta pronto!
